sensor/tests/resource/role/role_test.go

```diff
@@ -89,6 +91,24 @@ func (s *RoleDependencySuite) Test_PermutationTest() {
 	)
 }
 
+func (s *RoleDependencySuite) Test_ClusterRolePermutationTest() {
+	s.testContext.GetFakeCentral().ClearReceivedBuffer()
+	s.testContext.RunTest(
+		resource.WithResources([]resource.K8sResourceInfo{
+			NginxDeployment,
+			NginxClusterRole,
```

Comment:
There might be one issue with this: `ClusterRole` resources are not namespaced, and the test context library will always apply resources in a default namespace. [See this](https://github.com/stackrox/stackrox/blob/master/sensor/tests/resource/helper.go#L264). 

If tests are passing then it's probably not an issue. But maybe it's worth double checking that the behavior of applying a cluster role (+ binding) with namespace is the same as applying them globally.

Reply:
Double checked and the `ClusterRole`/`Binding` are created properly. The namespace gets ignored.